apps/vx-scan/frontend/test/helpers/mock_api_client.tsx

```diff
@@ -40,16 +46,83 @@ export const statusNoPaper: PrecinctScannerStatus = {
   ballotsCounted: 0,
 };
 
+type MockApiClient = Omit<
+  MockClient<Api>,
+  'getAuthStatus' | 'writeCardData'
+> & {
+  // Because this is polled so frequently, we opt for a standard jest mock instead of a
+  // libs/test-utils mock since the latter requires every call to be explicitly mocked
+  getAuthStatus: jest.Mock;
+  // Because the values passed to this are so complex, we opt for a standard jest mock instead of a
+  // libs/test-utils mock since the latter requires exact input matching and doesn't support
+  // matchers like expect.objectContaining
+  writeCardData: jest.Mock;
```

Comment:
I contemplated just providing the exact values, but they were hard to generate and also included timestamps. I know the timestamps can be worked around by mocking time, but this didn't feel worth the trouble